tests/network/vmi_passt.go

```diff
@@ -43,6 +45,9 @@ var _ = SIGDescribe("[Serial] Passt", func() {
 	var err error
 	var virtClient kubecli.KubevirtClient
 
+	BeforeEach(func() {
+		checks.SkipTestIfNoFeatureGate(virtconfig.PasstGate)
+	})
```

Comment:
nit you can ignore: would you add a new line here, to have the `BeforeEach` blocks separated ?

Reply:
nit: We can use already defined BeforeEach below.

Reply:
I kept the separate BeforeEach, since they are unrelated.